### PR TITLE
Issue/probablistic

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- add column "properties" to ForecastValue that can store probablistic data
- update save method
- add migration
- update blend method
- need to pin `pydantic=1.10.10` as there were some breaking changes - https://github.com/openclimatefix/nowcasting_datamodel/issues/205

https://github.com/openclimatefix/nowcasting_datamodel/issues/197

## How Has This Been Tested?

- updated save test to check these values
- normal CI

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: nowcasting_datamodel/models/convert.py [hunk 1]
```diff
@@ -97,6 +99,13 @@ def convert_df_to_national_forecast(
             expected_power_generation_megawatts=forecast_value.forecast_mw,
         ).to_orm()
         forecast_value_sql.adjust_mw = 0.0
+
+        if "forecast_mw_plevel_10" in forecast_values_df.columns:
```
Comment (peterdudfield) [inline nowcasting_datamodel/models/convert.py:103]: split into two if statements

File: nowcasting_datamodel/models/forecast.py [hunk 2]
```diff
@@ -310,6 +318,12 @@ class ForecastValue(EnhancedBaseModel):
         "The _ at the start means it is not expose in the API",
     )
 
+    _properties: dict = Field(
```
Comment (peterdudfield) [inline nowcasting_datamodel/models/forecast.py:321]: add commnent about mkaing new pydantic model in API with these

**Review by devsjc — APPROVED** [review]
I guess we're using a JSON column instead of columns per the fields in the JSON because they are expected to be inconsistent and numerous between different forecasts?

**Comment by peterdudfield** [comment]
> I guess we're using a JSON column instead of columns per the fields in the JSON because they are expected to be inconsistent and numerous between different forecasts?

Yea, and for the moment I think we will store `plevel_10` and `plevel_90` but I think this could change quite soon, so this keeps it a bit more flexible

**Pull request merged**